.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# Change import statement paths to path aliases
```

Comment:
To follow the recommendation by Github – https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view

```suggestion
# .git-blame-ignore-revs
# Change import statement paths to path aliases
```